priv/static/thesis-editor.js

```diff
@@ -1870,7 +1870,8 @@ if (!("classList" in document.createElement("_"))) {
 
```

Comment:
Ignore, skip to https://github.com/infinitered/thesis-phoenix/pull/26/files#diff-09f54a59088121d2a18a7de753676004